### ISSUE TITLE
Default Generated Form Fails in On Disk Project

### ISSUE THREAD

**Issue by JoelProminic** [Moonshine-IDE/Moonshine-IDE#872]
While testing #853, I noticed this error.  I think I have seen it before:
1. Create a new On Disk Project with File > New > Domino On Disk Project
2. Open nsfs/nsf-moonshine/odp/Forms/.  Note that `%projectname%.form` exists in this directory
3. Project > Build & Run with Apache Maven. BUG:  this fails with this error

    : [INFO] !MESSAGE Application error
    : [INFO] !STACK 1
    : [INFO] java.util.co
    : ncurrent.ExecutionException: java.lang.RuntimeException: Exception while importing element Forms/TestOnDiskProject_0629.form
    : [INFO] 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
    : [INFO] 	at java.util.concurrent.FutureTask.get(FutureTask.java:19
    : 2)
    : [INFO] 	at org.openntf.nsfodp.compiler.equinox.CompilerApplication.start(CompilerApplication.java:111)
    ...
    : [INFO] Caused by: com.darwino.domino.napi.DominoException: Exception importing Basic element Forms/TestOnDiskProject_0629.form: item element must contain at least one child element (lin
    : e=17, column=4)
    : [INFO] 	at org.openntf.nsfodp.compiler.ODPCompiler.importDxl(ODPCompiler.java:866)
    : [INFO] 	at org.openntf.nsfodp.compiler.ODPCompiler.lambda$9(ODPCompiler.java:592)
    : [INFO] 	... 33 more

I would like to either remove `%projectname%.form` or populate it so that it won't trigger the above errors.  Is this file associated with `src/main/formbuilder/%projectname%.dfb` or `src/main/formbuilder/%projectname%.dve`?  If it is the .dfb file, then this will probably not be overwritten in practice, since the user will choose a different form name.   If it is associated with the .dve, then it probably needs to be populated with a default value instead.

**Comment by rat-moonshine** [comment]
Tested on last stable v3.2.0, the said file properly generated with appropriate naming for me after a new project creation:
![image](https://user-images.githubusercontent.com/24370870/123904519-399b6200-d98e-11eb-8add-79ee0e940e90.png)

**Comment by feather812002** [comment]
Yeah, I can repduce this error.
It cause this :

> : [INFO] Caused by: com.darwino.domino.napi.DominoException: Exception importing Basic element Forms/TestOnDiskProject_0629.form: item element must contain at least one child element (lin
> : e=17, column=4)

That mean , the "item name=$BOdy" must contain at least one child in it.
So I add follow code : 
`<item name="$Body" sign="true">
   <richtext style="width:700px;height:700px;" class="flexHorizontalLayout flexHorizontalLayoutLeft flexHorizontalLayoutTop" direction="Horizontal" vdirection="Vertical"/>
  </item>`

And got it work fine .

So just add a "richtext" filed into the body item it will become fine.

**Comment by JoelProminic** [comment]
I tested @feather812002's update.  I confirmed that the error is resolved, though I do see this error if I try to open the form in the database.  That said, this form would be blank anyway, so it is not really useful anyway.

![image](https://user-images.githubusercontent.com/24532731/123987989-3f447800-d98d-11eb-9b10-26bff3a27dc1.png)

@feather812002 reported some concerns about the commit.   Let me know if this needs to be re-tested after some cleanup.
